modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
+
+[role="_abstract"]
+You must update the sensor and admission-control deployment .yaml files to set the `POD_NAMESPACE` environment variable.
+
+[NOTE]
+====
+If you are using Kubernetes, use `kubectl` instead of `oc` for the commands listed in this procedure.
+====
+
+.Procedure
+
+. Navigate to your secured cluster bundle.
+. Open the `sensor.yaml` file in an editor
+. Navigate to `spec -> template -> spec -> containers -> env`
+. Add the following entry under `env`:
++
+[source,yaml,subs=attributes+]
+----
+  - name: POD_NAMESPACE
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+----
+. Apply the modified deployment `sensor.yaml` file to the cluster
++
+[source,terminal,subs=attributes+]
+----
+$ oc -n stackrox apply -f sensor.yaml
+----
```

Comment:
Could we maybe craft a `kubectl patch` command, so that the users could copy-paste it?

Reply:
I hadn't heard of those before, I'll look into this and try to do it that way.

Reply:
It seems it should be `kubectl patch deployment sensor -p '{"spec":{"template":{"spec":{"containers":{"env":{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"fieldPath":"metadata.namespace"}}}}}}}}'` 

I'll test this on a cluster to be sure and then add it to the docs if it works.

Reply:
Great! Yes, please do a test on the current version 4.5

Reply:
I took a trip into the land of JSON patches, the syntax was a bit more difficult than I first thought. But the patch commands now work on 4.5, just tested them on a cluster.